### PR TITLE
Env update for Auth

### PR DESCRIPTION
**Pull request by Vedo-Kodecta**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:qLNLNxel0tiZebepW7pDgFkZLGOq+5ZFi/2uQPr1WSY=
```
Comment (ErminMaksumic) [inline .env.example:3]: Please remove this.

File: config/cors.php [hunk 2]
```diff
@@ -17,9 +17,9 @@
 
     'paths' => ['api/*', 'api', 'sanctum/csrf-cookie'],
 
-    'allowed_methods' => ['*'],
+    'allowed_methods' => ['POST', 'GET', 'OPTIONS', 'PUT', 'DELETE'],
```
Comment (ErminMaksumic) [inline config/cors.php:20]: Do we really need this? When we put *, that means ['POST', 'GET', 'OPTIONS', 'PUT', 'DELETE'].

File: config/cors.php [hunk 3]
```diff
@@ -17,9 +17,9 @@
...
 
-    'allowed_origins' => ['*'],
+    'allowed_origins' => ['http://localhost:3000', 'http://localhost:8000'],
```
Comment (ErminMaksumic) [inline config/cors.php:22]: See above comment.

File: config/cors.php [hunk 4]
```diff
@@ -17,9 +17,9 @@
...
```
Comment (ErminMaksumic) [inline config/cors.php:23]: Added some comments.

**Review by ErminMaksumic — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (Vedo-Kodecta) [reply .env.example:3]: This uses php artisan key:generate which is required if we are using sanctum:login from csrf-cookie generation. For more info here: https://laravel.com/docs/7.x/installation

File: config/cors.php (hunk 2 above)
Comment (Vedo-Kodecta) [reply config/cors.php:20]: Yes we need this for CORS, it common knowledge

File: config/cors.php (hunk 3 above)
Comment (Vedo-Kodecta) [reply config/cors.php:22]: see the answer above

**Review by ErminMaksumic — APPROVED** [review]

**Pull request merged**